doc/migrations/v0.26-v0.27.md

```diff
@@ -0,0 +1,182 @@
+# Migrating to the new API
+
+A migration guide for refactoring your application code from libp2p v0.26.x to v0.27.0.
+
+## Table of Contents
+
+- [Migrating to the new API](#migrating-to-the-new-api)
+  - [Table of Contents](#table-of-contents)
```

Comment:
I like your style, but do these two _really_ need to be included here?

Reply:
I'll remove, the TOC plugin I'm using auto adds them.